dockerfiles/alpine/Dockerfile

```diff
@@ -12,4 +14,4 @@ VOLUME /dist
 VOLUME /home/build
 WORKDIR /home/build
 USER build
-
+RUN git config --global --add safe.directory /src/icu
```

Comment:
duplicate?

Several of these files now have two of these lines.

Reply:
Fixed!